tests/payments.py

```diff
@@ -40,4 +41,20 @@ def test_create_payment_type(self):
         self.assertEqual(json_response["expiration_date"], "2024-12-31")
         self.assertEqual(json_response["create_date"], str(datetime.date.today()))
 
-    # TODO: Delete payment type
\ No newline at end of file
+    # TODO: Delete payment type
+    
+    def test_delete_payment(self):
+        payment = Payment()
+        payment.merchant_name = 'bob'
+        payment.account_number = '123456'
+        payment.customer_id = 1
+        payment.expiration_date = '2023-12-15'
+        payment.create_date = '2020-12-15'
+        payment.save()
+
+        self.client.credentials(HTTP_AUTHORIZATION='Token ' + self.token)
+        response = self.client.delete(f"/paymenttypes/{payment.id}")
+        self.assertEqual(response.status_code, status.HTTP_204_NO_CONTENT)
+
+        response = self.client.get(f"/paymenttypes/{payment.id}")
+        self.assertEqual(response.status_code, status.HTTP_500_INTERNAL_SERVER_ERROR)
```

Comment:
Does this test pass? It should return a 404 if the thing you're trying to get doesn't exist.

Reply:
it does pass. the create payment is throwing an error but it was throwing an error before i created the delete payment test. i was gonna address it in a separate ticket.

Reply:
![image](https://user-images.githubusercontent.com/62913995/107458704-36f0bb00-6b1a-11eb-970c-4557fe59bc15.png)


that's the only error but its for create payment. unless i'm over looking something.

Reply:
Just realized the 404 is returned from the destroy and the 500 you're asserting is from the retrieve, so nevermind!